cast-dssspell.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+set -e
+
+[[ "$(seth chain --rpc-url="$ETH_RPC_URL")" == "goerli" ]] || { echo "Please set a Goerli ETH_RPC_URL"; exit 1; }
+
+CHANGELOG=0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F
+MCD_ADM=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "MCD_ADM")")")
+MCD_GOV=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "MCD_GOV")")")
+MCD_IOU=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "MCD_IOU")")")
+BALANCE=$(seth call "$MCD_GOV" 'balanceOf(address)(uint256)' "$ETH_FROM")
+HAT=$(seth call "$MCD_ADM" 'hat()(address)')
+APPROVALS=$(seth call "$MCD_ADM" 'approvals(address)(uint256)' "$HAT")
+DEPOSITS=$(seth call "$MCD_ADM" 'deposits(address)(uint256)' "$ETH_FROM")
+HAT_THRESHOLD=$(seth --to-wei 100000 ETH)
+CONTI=1
+
+if [[ -z "$1" ]];
+then
+    echo "Please specify the Goerli Spell Address"
+else
+    if [[ "$APPROVALS" -ge "$DEPOSITS" ]] || [[ "$DEPOSITS" -lt "$HAT_THRESHOLD" ]];
+    then
+    GAP=$((HAT_THRESHOLD - DEPOSITS))
+    DELTA=$((APPROVALS - DEPOSITS + GAP))
```

Comment:
This doesn't work. For example, let's say you have:
```
approvals=80K
deposits=95K
```

Then DELTA will be `-10K` when it should be `5K`.

Reply:
`approvals` should be greater of equal to `deposits` in this conditional block.

Reply:
aa2bbeb should fix it